community/advanced_usage/artisan.md

```diff
@@ -0,0 +1,153 @@
+# Artisan CLI
+
+::: tip
+Artisan is a file located within your Pterodactyl installation directory. If you followed the official guide, it should be `/var/www/pterodactyl`.
+:::
+
+::: tip
```

Comment:
Put this right below the help command, since thats the first place it gets used.